submissions/api.py

```diff
@@ -709,6 +710,11 @@ def set_score(submission_uuid, points_earned, points_possible):
         points_earned (int): The earned points for this submission.
         points_possible (int): The total points possible for this particular student item.
 
+        score_creator (str): An optional field for recording who made this particular score
```

Comment:
Nit: "who made this particular score" reads a bit ambiguously to me - it could mean either:
  * The original learner, as in "Little Bobby made a score of 94 on his homework"
  * The grader, as in "Big Bobby made a score of 94; that is what will be assigned to Little Bobby's homework response"

Maybe "who gave this particular score"?